### ISSUE TITLE
Logo problems on linux

### ISSUE THREAD

**Issue by nico202** [goropikari/Notifier.jl#9]
https://github.com/goropikari/Notifier.jl/blob/a8bc064c454beb73ebe7aad4e221544317c337f3/src/linux.jl#L27

Hi, for some reason my libnotify does not support SVGs, so this command fails. It would be nice to be able to use a custom logo (as a workaround and as a feature).

Example (bash):
Working:
notify-send Julia "this is a message" -i /home/username/.julia/v0.6/Notifier/src/logo.png -t 3000 # (after creating the logo.png
notify-send Julia "this is a message" -i /home/username/.julia/v0.6/Notifier/src/nonexisting.png -t 3000 
notify-send Julia "this is a message"-t 3000
NOT working:
notify-send Julia "this is a message" -i /home/IITaudio/.julia/v0.6/Notifier/src/logo.svg -t 3000

**Comment by goropikari** [comment]
I have modified the program and the logo can be changed by keyword option.
https://github.com/goropikari/Notifier.jl/commit/f4b0e6a70ba1f86ff237ec5d00eed7a5da416131#diff-3e80156dcb91f49b0db0cf0947f0cc3a

Because I can't reproduce your error, could you test following code in your machine?

```julia
Pkg.update()
Pkg.checkout("Notifier", "logo_error")
using Notifier
notify("foo", logo="")
```

**Comment by nico202** [comment]
Yes now the notification appears, thanks!

**Comment by goropikari** [comment]
I merged #10 into master branch. I will delete logo_error branch, so please checkout master branch.

```julia
Pkg.update()
Pkg.checkout("Notifier", "master")
```
